qiskit_experiments/library/characterization/cr_hamiltonian.py

```diff
@@ -0,0 +1,424 @@
+        The net duration of pulse edges are approximated by
+
+        .. math::
+
+            \tau_{\rm edges} = \sqrt{2 \pi \sigma^2},
+
+        where the :math:`\sigma` is the sigma of Gaussian rising and falling edges.
+        Thus the net duration of the entire cross resonance pulse will become
+
+        .. math::
+
+            \tau = {\rm duration} - 2 r \sigma + \tau_{\rm edges},
+
+        where :math:`r` is the ratio of the actual edge duration to the sigma.
+
+    # section: reference
+        .. ref_arxiv:: 1 1603.04821
+
+    # section: tutorial
+        .. ref_website:: Qiskit Textbook 6.7,
+            https://qiskit.org/textbook/ch-quantum-hardware/hamiltonian-tomography.html
+    """
+
+    __analysis_class__ = CrossResonanceHamiltonianAnalysis
+
+    def __init__(
+        self, qubits: Tuple[int, int], durations: Iterable[float], unit: str = "dt", **kwargs
+    ):
+        """Create a new experiment.
+
+        Args:
+            qubits: Two-value tuple of qubit indices on which to run tomography.
+                The first index stands for the control qubit.
+            durations: The pulse durations to scan.
+            unit: The time unit of durations.
+            kwargs: Pulse parameters. See :meth:`experiment_options` for details.
+
+        Raises:
+            QiskitError: When ``qubits`` length is not 2.
+        """
+        super().__init__(qubits=qubits)
+
+        if len(qubits) != 2:
+            raise QiskitError(
+                "Length of qubits is not 2. Please provide index for control and target qubit."
+            )
+
+        self.set_experiment_options(durations=durations, unit=unit, **kwargs)
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default experiment options.
+
+        Experiment Options:
+            durations (np.ndarray): The length of the cross resonance tone.
+            unit (str): Time unit of durations.
+            amp (complex): Amplitude of the cross resonance tone.
+            amp_t (complex): Amplitude of the cancellation or rotary drive on target qubit.
+            sigma (float): Sigma of Gaussian rise and fall edges.
+            risefall (float): Ratio of edge durations to sigma.
+        """
+        options = super()._default_experiment_options()
+        options.durations = None
+        options.unit = "dt"
+        options.amp = 0.2
+        options.amp_t = 0.0
+        options.sigma = 64
+        options.risefall = 2
+
+        return options
+
+    def _define_decomposition(
+        self,
+        backend: Backend,
+        duration: circuit.Parameter,
+        sigma: float,
+    ) -> QuantumCircuit:
+        """Single tone cross resonance.
+
+        Args:
+            backend: The target backend.
+            duration: Parameter object representing a duration of cross resonance pulse.
+            sigma: Sigma of Gaussian edges.
+
+        Returns:
+            A circuit decomposition for the cross resonance to measure.
```

Comment:
```suggestion
            A circuit decomposition for the cross resonance pulse to measure.
```

"decomposition" sounds a little odd to me because usually I think of that as meaning breaking something down into parts but the circuit here is just the single CR pulse. Maybe just adding "operation" to the docstring makes it clear enough. I would consider if a name like `_build_cr_circuit` or something like that would be more clear than `_define_decomposition` when reading it in the context of the `circuits` method.

Reply:
Thanks, done in https://github.com/Qiskit/qiskit-experiments/pull/385/commits/51a31ef2324a03ffe42bd5d27985139bc28e5690